### ISSUE TITLE
Opening the app after leaving it open for a while refreshes page and logged out

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#52073]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:** needs reproduction
**Reproducible in production?:** needs reproduction
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @quinthar 
**Slack conversation** (hyperlinked to channel name): [quality](https://applause-ts.slack.com/archives/C05LX9D6E07/p1730828577047929)

## Action Performed:
1. Sign in to ND in chrome browser
2. Wait for the laptop to go to sleep mode
3. After a while open the laptop and go to ND app
## Expected Result:
Should reconnect automatically and user is able to browse normally
## Actual Result:
Browser tab was blank white page.  It immediately refreshed and it shows sign in p
In  application state showing this `passwordForm.error.fallback `error 
## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

![image (1)](https://github.com/user-attachments/assets/4e677cee-9419-47b2-b1be-1270dce55054)

  

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by isabelastisser** [comment]
@youssef-lr, I added the internal label. Can you please confirm that you're working on this? Thanks!

**Comment by neil-marcellini** [comment]
Hey @youssef-lr I think this might effectively be a duplicate of this issue [Got logged out of the app in web staging](https://github.com/Expensify/App/issues/51707#top). Please lmk what you think. I believe I found the root cause and can have a fix up soon.

**Comment by youssef-lr** [comment]
@neil-marcellini yeah that looks like the same issue here. Should we close this out?

**Comment by isabelastisser** [comment]
I'm closing this one based on the comments above.
